modules/apps/headless/headless-builder/headless-builder-web/src/main/resources/META-INF/resources/js/components/baseComponents/BaseAPIApplicationFields.tsx

```diff
@@ -50,6 +50,19 @@ export default function BaseAPIApplicationFields({
 		urlAutoFillInitialDisable ?? false
 	);
 
+	const baseURLEnterValidErrorMessage = Liferay.Language.get(
+		'please-enter-a-valid-url'
+	);
+
+	const baseURLPlaceholderText = Liferay.Language.get('automated-url');
+
+	const [baseURLContent, setBaseURLContent] = useState({
+		errorMessage: Liferay.Language.get(
+			'please-enter-a-title-so-we-can-create-an-url'
+		),
+		placeholder: baseURLPlaceholderText,
```

Comment:
I think that you could inline this

Reply:
Yes, you are right. This is reminiscent of a different implementation. I will fix it. Thanks!